ext/drwrap/drwrap.c

```diff
@@ -885,6 +885,11 @@ static dr_emit_flags_t
 drwrap_event_bb_insert(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst,
                        bool for_trace, bool translating, void *user_data);
 
+static dr_emit_flags_t
+drwrap_event_bb_insert_where(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst,
```

Comment:
Though this is not an API, it'll be helpful to add a comment for this and `drwrap_event_bb_insert`.

Reply:
Done.